Description: disable calls to mmap() on some arches
That is armel, armhf, mips, mipsel and powerpc, where mmap fails with EINVAL.
Author: Louis Bettens <louis@bettens.info>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: haskell-hashable-1.2.1.0/hashable.cabal
===================================================================
--- haskell-hashable-1.2.1.0.orig/hashable.cabal	2013-09-13 23:31:52.000000000 +0200
+++ haskell-hashable-1.2.1.0/hashable.cabal	2013-10-23 22:30:39.000000000 +0200
@@ -77,7 +77,7 @@
                      QuickCheck >= 2.4.0.1,
                      random == 1.0.*,
                      text >= 0.11.0.5
-  if !os(windows)
+  if !os(windows) && !(arch(arm) || arch(mips) || arch(mipsel) || arch(powerpc))
     Build-depends:   unix
     CPP-options:     -DHAVE_MMAP
     Other-modules:   Regress.Mmap
